maintainers/maintainer-list.nix

```diff
@@ -6686,6 +6686,16 @@
     githubId = 15373888;
     name = "Claudius Holeksa";
   };
+  ken-matsui = {
```

Comment:
It looks like this is your first contribution.

Put this file in a separate commit: `maintainers: add ken-matsui`.

Reply:
Thank you so much for your reviews! I will fix them :)